addons/mail/static/src/core/common/attachment_view.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.AttachmentView">
         <div t-if="state.thread.attachmentsInWebClientView.length > 0" class="o-mail-Attachment">
             <t t-if="state.thread.mainAttachment">
-                <h3 t-if="!state.thread.mainAttachment.isPdf" class="mt0 mb8 ps-2 text-muted float-end"><t t-esc="state.thread.mainAttachment.filename"/></h3>
+                <h3 t-if="!state.thread.mainAttachment.isPdf" class="mt0 mb8 ps-2 text-muted d-flex justify-content-center"> <div> <t t-esc="state.thread.mainAttachment.filename"/> </div> </h3>
```

Comment:
[`h3` elements expect phrasing content as children](https://html.spec.whatwg.org/multipage/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements). Perhaps you can replace the `div` with a `span`? Or couldn't you just avoid using an extra element? At first glance this `div` looks useless, but I guess you use it to center the text using flexbox? If you only use phrasing content as it is expected for the content of an `h3` element, I believe that you should be able to centre it simply with `text-center`